public/javascripts/sunrise.js

```diff
@@ -1,3 +1,8 @@
+// Focus email field in reset change password modal
+$("#forgot-password-modal").on('shown.bs.modal', function(){
+    $(this).find('#reset-email-input').focus();
```

Comment:
And it's not used anymore, I just forgot to remove it after we decided to not use modal dialogs ;-)